src/autocomplete_abstract.coffee

```diff
@@ -326,7 +330,12 @@ class AutocompleteAbstract
         @list
 
     handleEmptyList: ->
-        @close()
+        if @options.showNoResultsNotice
+            noResultNotice = $ '<div class=empty-list>No Result Found…</div>'
+            noResultNotice.on 'click', (e) => @close()
```

Comment:
Why did you decide to close it on click? I think it's better to just leave it open.